### PR TITLE
Feat: [Term Entry] C# Strings .PadLeft()

### LINKED ISSUE #5567: [Term Entry] C# Strings .PadLeft()
### Reason/inspiration (optional)

We would like a new entry on the `.PadLeft()` term under the Strings concept in C#. The entry should go in a new file under `content/c-sharp/concepts/strings/terms/padleft/padleft.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept
- An `Example` section that provides an example demonstrating the concept in use
- A `Codebyte Example` section that provides a codebyte example demonstrating the concept in use

Please refer to the [term entry template](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by SaiTeja-002**
### Description

I've added a new entry for C# string methods - `.PadLeft()`. This entry can be found at `content/c-sharp/concepts/strings/terms/padleft/padleft.md`.

### Issue Solved

closes #5567 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @SaiTeja-002 Please also add a ## Codebyte Example block at the end.

File: content/c-sharp/concepts/strings/terms/padleft/padleft.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        string paddingWithChar = str.PadLeft(totalLength, padChar);
+        Console.WriteLine($"Padded with '{padChar}' to length {totalLength}: '{paddingWithChar}'");
+
+        // No padding applied if the specified length is less than the string's length
+        int shorterLength = 5;
+        string noPadding = str.PadLeft(shorterLength);
+        Console.WriteLine($"Specified length ({shorterLength}) less than string length: '{noPadding}'");
+    }
+}
+```
```
Comment (mamtawardhani) [inline content/c-sharp/concepts/strings/terms/padleft/padleft.md:103]: Hey @SaiTeja-002 can we change the string here? As it is similar to the example block above

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Please change the codebyte example block, since that is quite similar to the example block.

**Review by mamtawardhani — APPROVED** [review]
Hey @SaiTeja-002 thank you for contributing to Codecademy Docs! 😄 

The entry looks good for a second round of review! 🚀

**Review by Sriparno08 — APPROVED** [review]
Looks good, @SaiTeja-002!

**Pull request merged**